readme.txt

```diff
@@ -109,6 +109,16 @@ Web Stories are powered by [AMP](https://amp.dev/), which adds some restrictions
 
 For the plugin's full changelog, please see [the Releases page on GitHub](https://github.com/google/web-stories-wp/releases).
 
+= 1.7.0 =
+
+**Release Date:** May 11, 2021.
+
+* New feature: save individual pages as custom page templates.
```

Comment:
"New feature: save a single page as a template for reuse across all your stories."?